### PR TITLE
Add My VISN/VAMC feature work

### PR DESCRIPTION
**Pull request by emyl3**
### JIRA issue link
[DM-2352](https://agile6.atlassian.net/browse/DM-2352)
[DM-2353](https://agile6.atlassian.net/browse/DM-2353)

## Description - what does this code do?
This PR adds the VISN and VAMC feature functionality.

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: spec/features/register_spec.rb [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-    fill_in('Password confirmation', with: 'Passwrrd')
-    click_button('Register')
-
-    expect(page).to be_accessible.according_to :wcag2a, :section508
-    expect(page).to have_content('Email must use @va.gov email address')
-    expect(page).to have_content('Email invalid')
-    expect(page).to have_content('Password must include 6 unique characters')
-    expect(page).to have_content('Password complexity requirement not met. Password must include 3 of the following: 1 uppercase, 1 lowercase, 1 digit and 1 special character')
-  end
+  # it 'should not let the user sign up with a weak password or without a va.gov email address' do
```
Comment (emyl3) [inline spec/features/register_spec.rb:4]: going to revert these changes

File: app/assets/stylesheets/dm/pages/_practice.scss [hunk 2]
```diff
@@ -1251,3 +1243,7 @@ tr.border_bottom td {
     }
   }
 }
+
```
Comment (emyl3) [inline app/assets/stylesheets/dm/pages/_practice.scss:1246]: removing this

File: app/assets/javascripts/facility_map.es6 [hunk 3]
```diff
@@ -0,0 +1,43 @@
+function initialize() {
```
Comment (emyl3) [inline app/assets/javascripts/facility_map.es6:1]: moved this to this file `app/assets/javascripts/va_facilities/map.es6`

File: app/controllers/va_facilities_controller.rb [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    end
+  end
+
+  private
+
+  def set_va_facility
+    @va_facility = VaFacility.friendly.find(params[:id])
+  end
+
+  def get_created_practices_categories(practices_array)
```
Comment (emyl3) [inline app/controllers/va_facilities_controller.rb:109]: removing this to use method in practice utils instead

**Review by bradjohnson92008 — APPROVED** [review]
LGTM.  Lets get this merged in to Master.
